k8s_spark/parquet.spark.helm/templates/secret.yaml

```diff
@@ -1,9 +1,11 @@
+{{ with .Values.aws_creds }}
 apiVersion: v1
 kind: Secret
 type: Opaque
 metadata:
-  name: {{ .Release.Name }}-auth
+  name: {{ $.Release.Name }}-auth
 data:
-  awskey: {{ if .Values.aws_creds.awskey }}{{ .Values.aws_creds.awskey | b64enc }}{{- else }}''{{- end }}
-  awssecret: {{ if .Values.aws_creds.awssecret }}{{ .Values.aws_creds.awssecret | b64enc }}{{- else }}''{{- end }}
-  awstoken: {{ if .Values.aws_creds.awstoken }}{{ .Values.aws_creds.awstoken | b64enc }}{{- else }}''{{- end }}
+  awskey: {{ .awskey | b64enc }}
+  awssecret: {{ .awssecret | b64enc }}
+  awstoken: {{ .awstoken | b64enc }}
```

Comment:
Got this error while trying to use these.
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: unknown object type "nil" in Secret.data.awstoken
```
I have to update to this:
```
  awskey: {{ if .awskey }}{{ .awskey | b64enc }}{{- else }}''{{- end }}
  awssecret: {{ if .awssecret }}{{ .awssecret | b64enc }}{{- else }}''{{- end }}
  awstoken: {{ if .awstoken }}{{ .awstoken | b64enc }}{{- else }}''{{- end }}
```

Is that a correct change? @briansaildrone

Reply:
I don't see any issue with that.  Empty string defaults make sense here, especially since token won't always be provided with the other values.   I've committed the change.